# 机器人Buff数据

Header header

# 是否正在补血
bool isHealing
# 是否枪口冷却加速
bool isShooterCoolingFaster
# 是否防御加成
bool isDefenseBuff
# 是否攻击加成
bool isAttackBuff